### PR TITLE
Don't add xml tags if blank or zero

### PR DESCRIPTION
**Pull request by mpidcock**
https://www.pivotaltracker.com/n/projects/2650644/stories/186899013

### CODE REVIEW

File: app/lib/submission_builder/ty2022/states/ny/documents/it201.rb [hunk 1]
```diff
@@ -94,7 +95,7 @@ def document
                   intake.dependents.each do |dependent|
                     xml.depInfo do
                       xml.DEP_CHLD_FRST_NAME claimed: dependent.first_name
-                      xml.DEP_CHLD_MI_NAME claimed: dependent.middle_initial
+                      xml.DEP_CHLD_MI_NAME claimed: dependent.middle_initial if dependent.middle_initial
```
Comment (mrotondo) [inline app/lib/submission_builder/ty2022/states/ny/documents/it201.rb:98]: I think .present? here would prevent empty strings from causing this to show

Comment (mpidcock) [reply app/lib/submission_builder/ty2022/states/ny/documents/it201.rb]: Good call, updated

File: app/lib/submission_builder/ty2022/states/ny/documents/it201.rb [hunk 2]
```diff
@@ -48,7 +47,9 @@ def document
                 xml.STD_ITZ_IND claimed: 1
                 add_non_zero_claimed_value(xml, :DED_AMT, :IT201_LINE_34)
                 add_non_zero_claimed_value(xml, :INC_B4_EXMPT_AMT, :IT201_LINE_35)
-                xml.EXMPT_NMBR claimed: calculated_fields.fetch(:IT201_LINE_36)
+                if calculated_fields.fetch(:IT201_LINE_36).positive?
```
Comment (jenny-heath) [inline app/lib/submission_builder/ty2022/states/ny/documents/it201.rb:50]: `.present?` is more straightforward

File: app/lib/submission_builder/ty2022/states/ny/documents/rtn_header.rb [hunk 3]
```diff
@@ -41,7 +41,9 @@ def document
                 end
                 xml.ACH_IND claimed: @submission.data_source.ach_debit_transaction? ? 1 : 2
                 xml.RFND_OWE_IND claimed: REFUND_OR_OWE_TYPES[@submission.data_source.refund_or_owe_taxes_type]
-                xml.BAL_DUE_AMT claimed: calculated_fields.fetch(:IT201_LINE_80)
+                if calculated_fields.fetch(:IT201_LINE_80).positive?
```
Comment (jenny-heath) [inline app/lib/submission_builder/ty2022/states/ny/documents/rtn_header.rb:44]: `.present?`

**Review by jenny-heath — APPROVED** [review]
just a couple of small tweaks, then good to go

**Pull request merged**